doc/classes/CanvasItem.xml

```diff
@@ -511,28 +511,28 @@
 		</method>
 	</methods>
 	<members>
-		<member name="light_mask" type="int" setter="set_light_mask" getter="get_light_mask">
+		<member name="light_mask" type="int" setter="set_light_mask" getter="get_light_mask" default="1">
 			The rendering layers in which this [CanvasItem] responds to [Light2D] nodes. Default value: [code]1[/code].
 		</member>
-		<member name="material" type="Material" setter="set_material" getter="get_material">
+		<member name="material" type="Material" setter="set_material" getter="get_material" default="null">
 			The material applied to textures on this [CanvasItem]. Default value: [code]null[/code].
 		</member>
-		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate">
+		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate" default="Color( 1, 1, 1, 1 )">
 			The color applied to textures on this [CanvasItem]. Default value: [code]Color(1, 1, 1, 1)[/code] (opaque "white").
 		</member>
-		<member name="self_modulate" type="Color" setter="set_self_modulate" getter="get_self_modulate">
+		<member name="self_modulate" type="Color" setter="set_self_modulate" getter="get_self_modulate" default="Color( 1, 1, 1, 1 )">
 			The color applied to textures on this [CanvasItem]. This is not inherited by children [CanvasItem]s. Default value: [code]Color(1, 1, 1, 1)[/code] (opaque "white")..
 		</member>
-		<member name="show_behind_parent" type="bool" setter="set_draw_behind_parent" getter="is_draw_behind_parent_enabled">
+		<member name="show_behind_parent" type="bool" setter="set_draw_behind_parent" getter="is_draw_behind_parent_enabled" default="false">
 			If [code]true[/code], the object draws behind its parent. Default value: [code]false[/code].
 		</member>
-		<member name="show_on_top" type="bool" setter="_set_on_top" getter="_is_on_top">
+		<member name="show_on_top" type="bool" setter="_set_on_top" getter="_is_on_top" default="null">
```

Comment:
Also bogus, this should be a boolean.

Reply:
We can probably grep for `type="bool".*default="null"` to find other similar occurrences (and same with other non-Object types).